noxfile.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import annotations
+
+import nox
+
+
+@nox.session
+def mkdocs(session: nox.Session):
+    session.install("-r", "docs-requirements.txt")
```

Comment:
Is it too much cowbell to use `requirements.in` and pip compile a lock file?

Reply:
Yeah, I think we can avoid lockfiles for now. This is a simple little docsite for the release docs, and I'd prefer not to have to deal with a bunch of noisy dep bump PRs.